src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -459,7 +458,7 @@ private long CreateBackupTask(PeriodicBackup periodicBackup, bool isFullBackup,
                     throw new BackupDelayException(
                         $"Failed to start Backup Task: '{periodicBackup.Configuration.Name}'. " +
                         $"The task exceeds the maximum number of concurrent backup tasks configured. " +
-                        $"Current value of Backup.MaxNumberOfConcurrentBackups is: {_serverStore.Configuration.Backup.MaxNumberOfConcurrentBackups:#,#;;0}")
+                        $"Current value of Backup.MaxNumberOfConcurrentBackups is: {_serverStore.MaxNumberOfConcurrentBackups:#,#;;0}")
```

Comment:
Message needs to be fixed.

`Current maximum number of concurrent backups is: 3` ?

Reply:
Done